producer.py

```diff
@@ -18,37 +18,52 @@
 """Update Thoth's database of CVEs."""
 
 import logging
-
+import asyncio
 import requests
 
+from version import __version__
+
+from thoth.common import __version__ as __common__version__
 from thoth.common import init_logging
+
 from thoth.solver.python import get_ecosystem_solver
 from thoth.python.exceptions import NotFound
+
+from thoth.storages import __version__ as __storages__version__
 from thoth.storages import GraphDatabase
 
-__version__ = "0.2.0"
+from thoth.messaging import __version__ as __messaging__version__
+from thoth.messaging import MessageBase
+from thoth.messaging.package_releases import CVEProvidedMessage
 
 init_logging()
+app = MessageBase().app
 
 _LOGGER = logging.getLogger("thoth.cve_update")
+__service_version__ = f"{__version__}+storages.{__storages__version__}.common.{__common__version__}.messaging.{__messaging__version__}"  # noqa: E501
+_LOGGER.info(f"Thoth-cve-update-producer v%s", __service_version__)
+
 _SAFETY_DB_URL = (
     "https://raw.githubusercontent.com/pyupio/safety-db/master/data/insecure_full.json"
 )
 
 
 def _list_cves() -> dict:
-    """Download pyup safety db and parse it."""
-    _LOGGER.debug("Downloading pyup CVE database")
+    """Download pypi safety db and parse it."""
```

Comment:
I think is correct pyup : https://github.com/pyupio/safety-db it's not pypi

Reply:
thanks, https://github.com/thoth-station/cve-update-job/pull/374/commits/95c13f97f108fa068010786d621ae72e0e7a87b0